cmd/start.go

```diff
@@ -94,36 +98,36 @@ to quickly create a Cobra application.`,
 	},
 }
 
-func setupLogging() *log.Entry {
-	log.SetOutput(os.Stdout)
-	if strings.ToLower(cliConfig.Logging.Format) == "json" {
-		log.SetFormatter(&log.JSONFormatter{})
+func setupLogging() logger.ILogger {
+	logger, err := logger.NewLogger("info", "", cliConfig.Logging.Format, false)
+	if err != nil {
+		panic(err)
 	}
-	log.SetLevel(log.InfoLevel)
-
-	return log.WithField("pkg", "cmd/start")
+	return logger
 }
 
 func initConfig() {
+	var log *zap.Logger
 	files, err := os.ReadDir("./")
 	if err != nil {
-		log.Fatal(err)
+		log.Fatal(err.Error())
 	}
 
 	for _, f := range files {
-		log.Println(f.Name())
+		// log.Info(f.Name())
+		fmt.Println(f.Name())
 	}
 	if cfgFile != "" {
 		// Use config file from the flag
 		yamlFile, err := os.ReadFile(cfgFile)
 		if err != nil {
-			log.WithError(err).WithField("cfgFile", cfgFile).Error("Unable to read config file")
+			log.With(zap.String("cfgfile", cfgFile)).Error("error while reading the config file", zap.Error(err))
```

Comment:
Why zap is used separately and not the wrapper ?

Reply:
Done